debian/control

```diff
@@ -16,13 +16,15 @@ Build-Depends: debhelper (>= 9),
                python-testtools,
                python-twisted-core (>= 16.0.0),
                python-txjuju,
+               python-txfixtures,
                python-yaml,
                wget
 
 Package: fake-juju
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends},
          juju-mongodb,
+         juju-mongodb3.2,
```

Comment:
Juju uses mongo 3.2 by default. Using it in fake-juju makes it closer to the real thing. I plan to drop juju-mongodb once we drop the old fake-juju code.